app/controllers/api/base_controller/authentication.rb

```diff
@@ -52,6 +56,15 @@ def require_api_user_or_token
         log_api_response
       end
 
+      #
+      # Attempt auth if desired, but flow through on failure
+      #
+      def optional_api_user_or_token
+        authenticate_user
+      rescue AuthenticationError => e
+        api_log_error("AuthenticationError: #{e.message} (on #{request.method}... ignoring)")
```

Comment:
wondering if we should log this more as a warning, since it's not really an error.

Reply:
Yeah, I just threw something in there for getting something pushed up, but definitely wasn't satisfied when I wrote it.

We can absolutely change it to a warning.  I will see if there is a method for this somewhere.  Happy to change anything else with this if you can think of anything better as well.